### PR TITLE
var show footer template load time en

### PR DESCRIPTION
**Pull request by avelino**

### CODE REVIEW

File: MAINTAINERS [hunk 1]
```diff
@@ -1,5 +1,6 @@
 Alexey Makhov <amakhov@avito.ru> (@makhov)
 Andrey Nering <andrey.nering@gmail.com> (@andreynering)
+Avelino <thiago@avelino.xxx> (@avelino)
```
Comment (lunny) [inline MAINTAINERS]: If you want to be as a maintainer, please See [CONTRIBUTING.md](CONTRIBUTING.md) and apply on gitter.im. Or maybe you want to add yourself to CONTRIBUTIORS.

**Review by lunny — CHANGES_REQUESTED** [review]

**Comment by appleboy** [comment]
@avelino Do you a member of Gitea organization? cc @lunny @tboerger

**Comment by avelino** [comment]
Sorry @lunny @appleboy 
I thought it might help keep the documentation (that's why I put my name)

**Review by appleboy — APPROVED** [review]

**Comment by appleboy** [comment]
LGTM @avelino It is ok.

**Review by lunny — APPROVED** [review]

**Comment by lunny** [comment]
LGTM

**Comment by lunny** [comment]
Thanks and seems this could be backport to v1.0. @avelino

**Pull request merged**